### PR TITLE
translation of dev/doc/style.txt in md

### PR DESCRIPTION
**Pull request by Villetaneuse**
I also corrected a couple typos.
Some parts do not make sense to me (the paragraph about parentheses needed around `try` in the end).

### CODE REVIEW

**Review by proux01 — APPROVED** [review]

File: dev/doc/style.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+  `match x with VarRef x -> bla | ConstRef x -> bli | _ -> blo` when
+  `GlobRef` is not open) it can be resolved in several ways:
+  + locally or globally open `GlobRef`
+  + type annotate `x : GlobRef.t` (where it is introduced, or in the `match` expression, whichever is nicer)
+  + annotate the first branch `GlobRef.VarRef x -> bla`
+    this last solution is not robust to branch reordering so should not be prefered
+
+## Coding methodology
+
+- no `try ... with _ -> ...` which catches even Sys.Break (Ctrl-C),
```
[the remark this answers is not in the document]
Comment (Villetaneuse) [reply dev/doc/style.md]: done, thanks for noticing this omission.

**Comment by jfehrle** [comment]
Putting this into md format is OK, but the document probably deserves thorough revision which would entail a certain amount of discussion (if it is to be taken seriously).

**Comment by Villetaneuse** [comment]
> Putting this into md format is OK, but the document probably deserves thorough revision which would entail a certain amount of discussion (if it is to be taken seriously).

Do you think this discussion should happen before merging this PR?

Feel free to create a thread on zulip. I'm really not an expert about this.

**Comment by jfehrle** [comment]
> Do you think this discussion should happen before merging this PR?

No.

**Comment by proux01** [comment]
@coqbot merge now

**Comment by proux01** [comment]
@coqbot merge now

**Pull request merged**